### ISSUE TITLE
grid_num does nothing?

### ISSUE THREAD

**Issue by MSCAU** [IonDen/ion.rangeSlider#423]
Hmm. Loving this but can't get grid_num to do anything. I end up with 3 ticks in each month instead of 1, except between Nov and Dec:

```
       <input type="text" id="slider_dates" name="slider_dates" value="" />

	var months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];

  	$("#slider_dates").ionRangeSlider({
			type: "double",
			grid: true,			
			grid_num: 1,
			from: 3,
			to: 6,
			values: months,
			hide_min_max: true
  	});
```

**Comment by IonDen** [comment]
Hi, grid_num does not work with values array. But i think i will fix that.

**Comment by MSCAU** [comment]
I see. How can I fix the ticks between Nov and Dec then?

**Comment by IonDen** [comment]
It is a bug.

**Comment by MSCAU** [comment]
Hi Den. Thanks for 2.17. When will you fix this issue?

**Comment by IonDen** [comment]
Fixed in 2.1.8

**Comment by sashsvamir** [comment]
2.3.0 stil not working `grid_num` when `values` array provided
